.github/workflows/disseminate.yml

```diff
@@ -40,12 +57,17 @@ jobs:
           workload_identity_provider: ${{ secrets.GOOGLE_WORKLOAD_ID_PROVIDER }}
           service_account: ${{ secrets.GOOGLE_SERVICE_ACCOUNT }}
 
-      - name: Run disseminator using Dockerhub image
+      - name: Run disseminator, save outputs, output errors to job summary
         run: |
-          /disseminator.py \
+          python disseminator.py \
             --work ${{ inputs.work-id }} \
-            --platform ${{ inputs.platform }} \
-            > ${{ inputs.work-id }}
+            --platform ${{ inputs.platform }} > ${{ inputs.work-id }} 2> /tmp/error.log || STATUS=$?
+          ERROR_LOG=$(grep '^ERROR:' /tmp/error.log)
```

Comment:
This approach means we lose access to any non-`ERROR`-level log lines, as they're no longer being printed to the Actions console log, and then they're being discarded when choosing what to print to the step summary.

One option might be to use `tee` to duplicate the output instead of piping it.